ui/app/partials/alerts.html

```diff
@@ -2,14 +2,20 @@
 	<fieldset>
 		<legend>Filter</legend>
 
-		<label>Receivers</label>
-		<select name="select-multi" class="width-2" ng-model="receivers" multiple="multiple"
-			ng-options="r for r in allReceivers track by r">
-		</select>
+		<section>
+			<label>Receivers</label>
+			<select name="select-multi" class="width-2" ng-model="receivers" multiple="multiple"
+				ng-options="r for r in allReceivers track by r">
+			</select>
+		</section>
+
+		<section>
+			<label class="checkbox"><input type="checkbox" ng-model="hideSilenced"> Hide silenced</label>
+		</section>
 	</fieldset>
 </form>
 
-<div id="alert-groups">
+<div id="alert-groups" ng-class="{'hide-silenced': hideSilenced}">
```

Comment:
Ah, the nested nature of this wasn't apparent to me. All makes sense then, I guess keep it like it is.